docs/searching_documents.rst

```diff
@@ -0,0 +1,155 @@
+Searching Documents
+===================
+
+Search is the primary operation of Elasticsearch. The client supports all kinds
+of searches supported by the Elasticsearch REST API. There are two different
+ways to search.
+
+URI Search
+----------
+
+This kinds of search uses a query string, which internally translates to a URI
+Search. Not all search options are exposed while using this mode. However,
```

Comment:
`Not all search options are exposed while using this mode.` this is weirdly wrong. This is actually using a specific mode [query string](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html), which can be very powerful. I suggest rewriting the paragraph to be more clear. Be careful with mixing the terms `kind` and `mode`